lib/editor/components/EditorInput.js

```diff
@@ -118,6 +118,33 @@ export default class EditorInput extends Component<Props> {
     }
   }
 
+  _renderRouteTypeOptions = (field: any) => {
+    const standardTypes = []
+    const extendedTypes = []
+    const routeTypes = ['Standard', 'Extended']
+    const fieldOptions = field.options
+    for (const option in fieldOptions) {
+      if (fieldOptions[option].group === 'Standard') {
+        standardTypes.push(fieldOptions[option])
+      } else {
+        extendedTypes.push(fieldOptions[option])
+      }
+    }
```

Comment:
This could be simplified:
```js
const standardTypes = Object.values(fieldOptions).filter(fieldOption => fieldOption.group === 'Standard')
// same for extendedTypes
```

Reply:
Even better, in the `return` statement, try this:
```js
return routeTypes
.map(type => Object.values(fieldOptions).filter(/*see above*/))
.map(routeTypes => {
  <optgroup>
    {routeTypes.map(routeType => /*what you already have for each std/extended types*/)}
  </optgroup>
})

Reply:
Nice, I like it! Thanks Binh